src/app/components/FileListItem/__tests__/FileListItem.test.js

```diff
@@ -0,0 +1,186 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import '@testing-library/jest-dom'
+import userEvent from '@testing-library/user-event'
+
+import FileListItem from '../FileListItem'
+import DownloadItem from '../../DownloadItem/DownloadItem'
+
+import downloadStates from '../../../constants/downloadStates'
+import { ElectronApiContext } from '../../../context/ElectronApiContext'
+
+jest.mock('../../DownloadItem/DownloadItem')
+
+const file = {
+  id: 123,
+  downloadId: 'mock-download-id',
+  filename: 'mock-file.png',
+  state: downloadStates.active,
+  percent: 42,
+  createdAt: 691690191124,
+  timeStart: 1691690280763,
+  timeEnd: null,
+  receivedBytes: 61587289,
+  totalBytes: 61587289,
+  remainingTime: 0
+}
+
+const setup = (overrideProps = {}) => {
+  const cancelDownloadItem = jest.fn()
+  const copyDownloadPath = jest.fn()
+  const openDownloadFolder = jest.fn()
+  const restartDownload = jest.fn()
+  const showWaitingForEulaDialog = jest.fn()
+  const showWaitingForLoginDialog = jest.fn()
+  const startReportingFiles = jest.fn()
```

Comment:
```suggestion
  const startReportingFiles = jest.fn()
```
We can remove this mocked function its not called in `fileListItem`